src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/services/SQSService.kt

```diff
@@ -1,42 +1,49 @@
 package uk.gov.justice.digital.hmpps.riskprofiler.services
 
-import com.amazonaws.services.sqs.AmazonSQSAsync
+import com.fasterxml.jackson.core.JsonProcessingException
 import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.databind.util.StdDateFormat
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.module.kotlin.KotlinModule
-import org.springframework.beans.factory.annotation.Qualifier
-import org.springframework.beans.factory.annotation.Value
-import org.springframework.cloud.aws.core.env.ResourceIdResolver
-import org.springframework.cloud.aws.messaging.core.QueueMessageChannel
-import org.springframework.cloud.aws.messaging.core.QueueMessagingTemplate
-import org.springframework.messaging.converter.MappingJackson2MessageConverter
+import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
+import software.amazon.awssdk.services.sqs.SqsAsyncClient
+import software.amazon.awssdk.services.sqs.model.SendMessageRequest
 import uk.gov.justice.digital.hmpps.riskprofiler.model.RiskProfileChange
+import uk.gov.justice.hmpps.sqs.HmppsQueueService
+import uk.gov.justice.hmpps.sqs.MissingQueueException
+import java.util.function.Consumer
 
 @Service
 class SQSService(
-  @Qualifier("awsSqsClient") amazonSqs: AmazonSQSAsync,
-  @Value("\${sqs.rpc.queue.url}") queueUrl: String
+  hmppsQueueService: HmppsQueueService,
+  private val objectMapper: ObjectMapper
 ) {
-  private val queueTemplate: QueueMessagingTemplate
-  private val amazonSqs: AmazonSQSAsync
-  private val queueUrl: String
+
+  private final val riskProfileChangeQueueSqsClient: SqsAsyncClient
+
+
+  init {
+    val riskProfileChangeQueue = hmppsQueueService.findByQueueId("risk_profiler_change") ?: throw MissingQueueException("Could not find topic riskprofilechangequeue")
```

Comment:
@steverendell Im not entirely sure how this matches up with the properties defined in valiues-dev? Is the physial queue name risk_profiler_change? or is it an alias stored in values-dev?